package.json

```diff
@@ -61,6 +61,7 @@
     "react": "^16.13.1",
     "react-apollo": "^3.1.3",
     "react-apollo-hooks": "^0.5.0",
+    "react-content-loader": "^6.0.1",
```

Comment:
hds supports small spinner, I think we should use that:  https://hds.hel.fi/components/spinner what do you think? It's best to ask about it from Tomi.
If we start to use hds spinner, we can delete our own spinner component completely, just remember then to put `data-testid="loading-spinner"` on it or many tests will fail.

Reply:
Tomi said that we need skeleton, spinner is not that good in that case. So i made a demo for him and he approved now.